yoda/gas.go

```diff
@@ -68,20 +72,47 @@ func getTxByteLength(msgs []sdk.Msg) uint64 {
 }
 
 func getRequestMsgByteLength(f FeeEstimationData) uint64 {
-	size := baseRequestSize
-	size += uint64(len(f.callData))
-	size += uint64(f.askCount) * addressSize
-	size += uint64(len(f.clientID))
-
-	for _, r := range f.rawRequests {
-		size += baseRawRequestSize + uint64(len(r.calldata))
+	var requestedValidator []sdk.ValAddress
+	for i := 0; i < int(f.askCount); i++ {
+		addr := []byte{200, 199, 198, 197, 196, 195, 194, 193, 192, 191, 190, 189, 188, 187, 176, 165, 154, 143, 132, 181}
```

Comment:
What are these bytes represent?

Reply:
I am trying to simulate validator addresses, but the current one is not quite good enough. Is there any suggestion?

Reply:
I think we can use the same code. We want to estimate it

Reply:
OK, reverted to previous code and adjust constants instead.